DotNet/Shared/Application/Repositories/Interfaces/IPersistenceRepository.cs

```diff
@@ -2,12 +2,12 @@
 
 public interface IPersistenceRepository<T>
 {
-    void Add(T entity);
-    Task AddAsync(T entity, CancellationToken cancellationToken = default);
-    T Get(string id);
-    Task<T> GetAsync(string id, CancellationToken cancellationToken = default);
-    T Update(T entity);
-    Task<T> UpdateAsync(T entity, CancellationToken cancellationToken = default);
-    void Delete(string id);
-    Task DeleteAsync(string id, CancellationToken cancellationToken = default);
+    public T Add(T entity);
```

Comment:
I don't think access modifiers are necessary for a interface. Just by being an interface, these methods are public by default.
https://stackoverflow.com/a/6040815